File: php/ping.php
<?php
function ping($ip)
{
    exec("ping -c 3 " . $ip, $output, $result);
    if ($result == 0)
        echo "LINK UP";
    else
        echo "LINK DOWN";
}
if (count($argv) < 1) {
    $ip = $argv[1];
    ping($ip);
} else {
    $ip = 'google.com';
    ping($ip);
}

File: python/ping.py
import os
import sys

def validaPing(ip):
    result = os.system("ping -c 1 " + ip)
    if result == 0:
        print('\033[32m'+'LINK UP'+'\033[0;0m')
    else:
        print('\033[31m'+'LINK DOWN '+'\033[0;0m')
if len(sys.argv) > 1:
    validaPing(sys.argv[1])
else:
    ip = input('Digite a url')
    validaPing(ip)
